packages/siri-sx-generator/test/siriSXUploadToDynamo.ts

```diff
@@ -1,38 +1,105 @@
-import { PutCommand } from "@aws-sdk/lib-dynamodb";
+/* eslint-disable no-console */
+import { BatchWriteCommand } from "@aws-sdk/lib-dynamodb";
 import { randomUUID } from "crypto";
-import { testDisruptionsJson } from "./testData";
+import { baseDisruptionJson } from "./testData";
 import { getDdbDocumentClient } from "../util/awsClient";
 
-const [stageName] = process.argv.slice(2);
+const [stageName, itemsToCreate] = process.argv.slice(2);
```

Comment:
The value for `itemsToCreate` is not configured to be passed from the script `publishJsonToDynamo.sh`

Reply:
It is - usage is like this: 

npm run dynamo-publish-json -- rob 100

Reply:
I think @alvak1 is referring to the script in the scripts folder which is a convenience wrapper around the npm command, it probably just needs updating to something like
```bash
#!/usr/bin/env bash

### Get the project root directory
cdd_root_folder_path=$(git rev-parse --show-toplevel)

stage_name=$(cat ${cdd_root_folder_path}/.sst/stage)
num_disruptions=$1

cd $cdd_root_folder_path/packages/siri-sx-generator

npm run dynamo-publish-json $stage_name $num_disruptions
```

Reply:
Done